plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/SQLServerUtils.java

```diff
@@ -342,6 +341,30 @@ public static String changeCreateToAlterDDL(
         return ddl;
     }
 
+
+    @NotNull
+    public static String getObjectDefinitionFunction(
```

Comment:
It's worth explaining what this function returns, e.g. a `select` statement with a single column, etc. And don't forget to update its name to reflect its behavior.